package.json

```diff
@@ -106,6 +106,7 @@
     "shelljs": "^0.8.2",
     "stylelint": "^9.2.0",
     "stylelint-config-terra": "^1.0.0",
+    "terra-base": "3.0.0",
```

Comment:
Why is this locked to 3.0.0?

Reply:
Ah, we should add a caret in front. `^3.0.0`

Reply:
That was already like that in the disclosure-manager package. I fixed it :)